### DISCUSSION TITLE
Hide the Import Export Reset buttons?

### DISCUSSION THREAD

**Discussion by MrKuenning** [ilian6806/stable-diffusion-webui-state#56] [Q&A] [answered]
I wanted to start by saying this is one of my favorite extensions for A1111, I find it basically essential for my workflows.

I was wondering if the was anyway to hide the Import Export Reset buttons?  They cause my mobile view to line-wrap.

**Comment by ilian6806** [comment, marked as answer]
In the last update, you have the option to hide all buttons individually. :)
